### PR TITLE
[Refactor] Drop RG param in getXXXID()

### PR DESCRIPTION
**Pull request by lzhecheng**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
[Refactor] Drop RG param in getXXXID(). Resource group name can be got from az.getLoadBalancerResourceGroup().
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/provider/azure_standard.go [hunk 1]
```diff
@@ -69,31 +69,31 @@ func (az *Cloud) getAvailabilitySetID(resourceGroup, availabilitySetName string)
 }
 
 // returns the full identifier of a loadbalancer frontendipconfiguration.
-func (az *Cloud) getFrontendIPConfigID(lbName, rgName, fipConfigName string) string {
+func (az *Cloud) getFrontendIPConfigID(lbName, fipConfigName string) string {
 	return fmt.Sprintf(
 		consts.FrontendIPConfigIDTemplate,
 		az.getNetworkResourceSubscriptionID(),
-		rgName,
+		az.getLoadBalancerResourceGroup(),
```
Comment (MartinForReal) [inline pkg/provider/azure_standard.go:76]: getFrontendIPConfigID will rely on az config, the function will return different value if the config is changed.

Comment (lzhecheng) [reply pkg/provider/azure_standard.go:76]: RG comes from `az.getLoadBalancerResourceGroup()` in all current usages. Added a new feature `XXXWithRG()` to allow using specific RG.
Updated.

Comment (MartinForReal) [reply pkg/provider/azure_standard.go]: LGTM

**Pull request merged**